src/upnp_xml.cc

```diff
@@ -471,31 +471,31 @@ std::pair<std::string, bool> UpnpXMLBuilder::renderItemImage(const std::string&
     return {};
 }
 
-std::pair<std::string, bool> UpnpXMLBuilder::renderSubtitle(const std::string& virtualURL, const std::shared_ptr<CdsItem>& item)
+std::pair<std::string, bool> UpnpXMLBuilder::renderSubtitle(const std::string& virtualURL, const std::shared_ptr<CdsItem>& item, const std::unique_ptr<Quirks>& quirks)
```

Comment:
Maybe just pass a reference to the Quirks here, not sure it makes sense to pass a reference to a unique_ptr

Reply:
Sometimes you do not see the obvious

Reply:
Needed to roll it back, because quirks are nullable. Crashed in tests

Reply:
Pointers work